### PR TITLE
Add ability to select AWS profile

### PR DESCRIPTION
**Pull request by bronislav**
`aws-cli` allows to setup named credential pairs which is quite useful. Let's utilize this cool feature.

### CODE REVIEW

**Comment by justinsb** [comment]
I use this feature, but I just set AWS_DEFAULT_PROFILE.  Are you doing something funky where this won't work?

I agree that profiles are a very useful feature, and we should document AWS_DEFAULT_PROFILE and/or merge this!

**Comment by bronislav** [comment]
@justinsb, I didn't know that `aws cli` supports direct profile switching using environment variables. I will add this to the documentation instead of changing script.

File: cluster/aws/util.sh [hunk 1]
```diff
@@ -36,8 +36,8 @@ esac
 AWS_REGION=${ZONE%?}
 
 export AWS_DEFAULT_REGION=${AWS_REGION}
-AWS_CMD="aws --output json ec2"
-AWS_ELB_CMD="aws --output json elb"
+AWS_CMD="aws --profile ${AWS_CMD_PROFILE:-default} --output json ec2"
+AWS_ELB_CMD="aws --profile ${AWS_CMD_PROFILE:-default} --output json elb"
```
Comment (justinsb) [inline cluster/aws/util.sh]: I don't think we need these any more (right?)

File: docs/getting-started-guides/aws.md [hunk 2]
```diff
@@ -6,6 +6,11 @@
 2. Install and configure [AWS Command Line Interface](http://aws.amazon.com/cli)
 3. You need an AWS [instance profile and role](http://docs.aws.amazon.com/IAM/latest/UserGuide/instance-profiles.html) with EC2 full access.
 
+NOTE: This script use default AWS profile by default. You may explicitly set AWS profile to use using `AWS_DEFAULT_FROFILE` environament variable:
```
Comment (justinsb) [inline docs/getting-started-guides/aws.md]: Typo: s/FROFILE/PROFILE/g

**Comment by justinsb** [comment]
Some fixes needed, but looks great if those are done!  Thanks for the fix!

**Comment by bronislav** [comment]
@justinsb Fixed. And rebased. Ready to merge!

**Comment by bronislav** [comment]
@justinsb Any updates?

File: docs/getting-started-guides/aws.md [hunk 3]
```diff
@@ -19,6 +19,11 @@ Getting started on AWS EC2
 2. Install and configure [AWS Command Line Interface](http://aws.amazon.com/cli)
 3. You need an AWS [instance profile and role](http://docs.aws.amazon.com/IAM/latest/UserGuide/instance-profiles.html) with EC2 full access.
 
+NOTE: This script use 'default' AWS profile by default. You may explicitly set AWS profile to use using `AWS_DEFAULT_PFROFILE` environament variable:
```
Comment (mbforbes) [inline docs/getting-started-guides/aws.md]: I think this should be "This script use**s the** 'default'..."

And also "using **the** `AWS_DEFAULT_PROFILE` ..."

File: docs/getting-started-guides/aws.md [hunk 4]
```diff
@@ -19,6 +19,11 @@ Getting started on AWS EC2
...
+```bash
```
Comment (mbforbes) [inline docs/getting-started-guides/aws.md]: I also prefer empty lines before code blocks, but it looks like we're not great about that in general, so it's your call.

**Comment by mbforbes** [comment]
@bronislav I'm so sorry that this PR got lost for so long! If you'd be kind enough to address my little nits we can shepherd this in ASAP. Thanks/sorry again.

**Comment by bronislav** [comment]
@mbforbes Fixed.

**Comment by mbforbes** [comment]
Shippable almost certainly a flake. rerunning....

**Comment by mbforbes** [comment]
@bronislav I'm wondering if you caught this at a time when shippable was busted? Could you please rebase from head and try pushing again so we can see if Shippable passes?

File: docs/getting-started-guides/aws.md [hunk 5]
```diff
@@ -51,6 +51,13 @@ Getting started on AWS EC2
 2. Install and configure [AWS Command Line Interface](http://aws.amazon.com/cli)
 3. You need an AWS [instance profile and role](http://docs.aws.amazon.com/IAM/latest/UserGuide/instance-profiles.html) with EC2 full access.
 
+NOTE: This script use the 'default' AWS profile by default.
+You may explicitly set AWS profile to use using the `AWS_DEFAULT_PFROFILE` environament variable:
```
Comment (justinsb) [inline docs/getting-started-guides/aws.md]: If you fix the typos here (_PFROFILE -> _PROFILE, environament -> environment), you might persuade shippable to rerun the test :-)

Comment (bronislav) [inline docs/getting-started-guides/aws.md]: Thanks for catching this.

**Comment by bronislav** [comment]
@mbforbes  Finally :) All checks are passed.

**Comment by justinsb** [comment]
Thanks @bronislav and sorry this took so long!

**Pull request merged**